packages/web-pkg/src/services/loadingService.ts

```diff
@@ -67,6 +67,16 @@ export class LoadingService {
       active: false,
       ...(!indeterminate && { state: { total: 0, current: 0 } })
     }
+
+    /**
+     * Add a task only if no other tasks are in progress.
```

Comment:
What does this line of the comment-block do? can't you remove it? The other two lines read like that's already everything I need to know...

Reply:
I asked chad gpt to make it a easy to read as possibly :D